search_that_hash/cracker/sth_mod/sth.py

```diff
@@ -82,13 +85,10 @@ def append_sth(self):
 
                 for type in self.config["hashes"][list(hash.keys())[0]]:
                     types.append(type["name"])
-                    if len(types) == 4:
+                    if len(types) == 5:
                         break  # We dont want to add all 100 possible types

```

Comment:
See this is good, you should explain the `5` with a comment like that. Maybe add that comment before `if len(types) == 5:`